qt/aqt/browser/table/table.py

```diff
@@ -358,11 +363,17 @@ def _update_font(self) -> None:
                 bsize = t.get("bsize", 0)
                 if bsize > curmax:
                     curmax = bsize
-        self._view.verticalHeader().setDefaultSectionSize(curmax + 6)
+
+        assert self._view is not None
+        vh = self._view.verticalHeader()
+        assert vh is not None
+        vh.setDefaultSectionSize(curmax + 6)
 
     def _setup_headers(self) -> None:
+        assert self._view is not None
         vh = self._view.verticalHeader()
-        hh = self._view.horizontalHeader()
+        assert vh is not None
```

Comment:
Yeah, I originally had the asserts the other way, but I kept getting an AssertionError on these asserts for the vertical header. For some reason, the `vh` object was being evaluated as falsy. So I decided to change all the asserts to be `is not None` instead in case that behavior was the same for any other Qt objects.